types/node/index.d.ts

```diff
@@ -5821,3 +5821,735 @@ declare module "async_hooks" {
      */
     export function createHook(options: HookCallbacks): AsyncHook;
 }
+
+declare module "http2" {
+    import * as events from "events";
+    import * as fs from "fs";
+    import * as net from "net";
+    import * as stream from "stream";
+    import * as tls from "tls";
+    import * as url from "url";
+
+    export interface Headers {
+        [headerField: string]: string | string[];
+    }
```

Comment:
maybe we could reuse `Http IncomingHttpHeaders` interface here?
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/index.d.ts#L789-L791